src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -152,6 +204,21 @@ export default class PatientViewMutationTable extends MutationTable<IPatientView
         this._columns[MutationTableColumnType.MRNA_EXPR].shouldExclude = ()=>{
             return (!this.props.mrnaExprRankMolecularProfileId) || (this.getSamples().length > 1);
         };
+
+        this._columns[MutationTableColumnType.CLONAL].shouldExclude = ()=>{
+            return !this.hasCcfMCopies;
+        };
+
+        this._columns[MutationTableColumnType.CANCER_CELL_FRACTION].shouldExclude = ()=>{
+            return !this.hasCcfMCopies;
+        };
+
+        // hide if multiple samples (same as Copy Number column)
+        // included because Mutant copies should only be shown in conjunction with Copy Num
```

Comment:
? I dont understand the comment. Would you mind rephrasing?

Reply:
think this was leftover from the first round of development when mutant copies wasn't shown on the patient-level view. removing this comment now!